SampleApp/package.json

```diff
@@ -30,6 +32,7 @@
     "cordova-plugin-android-permissions": "^1.1.5",
     "cordova-plugin-androidx-adapter": "^1.1.3",
     "cordova-plugin-badge": "^0.8.8",
-    "cordova-plugin-device": "^2.1.0"
+    "cordova-plugin-device": "^2.1.0",
+    "Verint-XM": "file:../../verint-xm-cordova-plugin"
```

Comment:
Was this meant to be committed?

Reply:
I hadn't thought it mattered too much, but now I'm remembering our chat. I'll remove it.